Library/Homebrew/brew.sh

```diff
@@ -883,6 +883,30 @@ then
   fi
 fi
 
+if [[ -n "${HOMEBREW_DEVELOPER}" || -n "${HOMEBREW_DEVELOPER_COMMAND}" ]]
+then
+  # Always run with Sorbet for Homebrew developers or Homebrew developer commands.
+  export HOMEBREW_SORBET_RUNTIME="1"
+fi
+
+# NO_SORBET_RUNTIME_COMMANDS are currently failing with Sorbet for homebrew/core.
+# TODO: fix this and remove this if block.
+if [[ -n "${HOMEBREW_SORBET_RUNTIME}" ]]
+then
+  NO_SORBET_RUNTIME_COMMANDS=(
+    audit
```

Comment:
Having `audit` run with `sorbet-runtime` was the primary reason for enabling this for developer commands in the first place. Might as well revert it completely again.

Reply:
Seemed fine the last time it was included: #15369 

Also, not convinced that this no longer has benefits that makes it worth enabling, even if we exclude `brew audit`.

Reply:
> Seemed fine the last time it was included: #15369 

https://github.com/Homebrew/brew/pull/15369 was merged so fast I didn't even see it.

`lsyncd` should be fixed instead of us having to re-re-re-re-re-re-revert these PRs constantly.

Reply:
Doesn't the version merging PR cover some of that?

If not, I can have a look.

Reply:
> Also, not convinced that this no longer has benefits that makes it worth enabling

I agree. This is incremental progress. That's better than turning off Sorbet for everything.

Reply:
> Doesn't the version merging PR already cover that?

@Bo98 no, that PR may only fix it by avoiding/hiding the error, the proper fix would be to make `SimulateSystem` support `MacOS.version` to actually get a proper version instead of `NULL`.

Reply:
The formula is alreaady designed to have a fallback if `MacOS.version` doesn't rerturn a value in the list. So `NULL` should be fine for that particular formula, once the internal erroring of `NULL` is fixed.